src/UI/Button.elm

```diff
@@ -245,7 +245,7 @@ buttonPadding ((Button { body } _) as btn) =
             Element.paddingXY 31 15
 
         BodyIcon _ ->
-            Element.paddingXY 9 15
+            Element.paddingXY 9 13
```

Comment:
Why this was changed?

Reply:
Look at docs, icon-buttons were rendering taller than text button, while on Sketch they're actually smaller (48px vs 40px - was gonna ask you about this anyway)

Reply:
Sufyian said 48px of height! This makes it!